dv/verilator/simutil_verilator/cpp/verilator_sim_ctrl.cc

```diff
@@ -295,6 +295,7 @@ bool VerilatorSimCtrl::MemWrite(const std::string &name,
   if (type == kMemImageUnknown) {
     std::cerr << "ERROR: Unable to detect file type for: " << filepath
               << std::endl;
+    // Continuing for more error messages
```

Comment:
Hm, maybe we want to return false here already?

Reply:
I did that at first, but then I liked the error messages that followed as it gave the name of the current memory region. It will exit anyway as the type is not set, but returning early and not rely on another check would be better. As a trade-off I added the comment.